### DISCUSSION TITLE
Nested Pipe functions

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [selfrefactor/rambda#659] [Q&A]
Sometimes there is a case where you want to have another function determining the predicate. Is there a nice way to use pipe with this?
A bit of pseudo code
```
pipe(
  filter(
    propSatisfies((item) => item.something == true),
    items
  ),
map((item) => item.value *2)
)

**Comment by selfrefactor** [comment]
I will need a bit more precise code, as I want to evaluate Typescript effect of any R.pipe change. Also, TS for R.pipe is far from ideal so any change need to be carefully introduced.

**Reply by sschneider-ihre-pvs** [reply]
that was actually pretty much the actual code :D I just wanted to know if there is maybe a nicer way to write it within a pipe

**Reply by selfrefactor** [reply]
I see, then there is not much nicer way to express your intention.
